classes/plugin.class.php

```diff
@@ -623,6 +623,20 @@ function get_product_meta_all() {
 		return $res;
 	}
 
+	/**
+	 * Get the count of all the created PPOM Group fields.
+	 * 
+	 * @return int - The number of group fields in the database.
+	 */
+	public static function get_product_meta_count() {
+		global $wpdb;
+
+		$qry = 'SELECT COUNT(*) FROM ' . $wpdb->prefix . PPOM_TABLE_META;
```

Comment:
@Soare-Robert-Daniel the cap on 100 was mentioned specifically to limit the query to count everything. So we can add a `limit 100` to avoid counting everything.